docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: "2"
+
+services:
+  web:
+      build: .
+      command: gunicorn -w 5 -b 0.0.0.0:8000 --log-level info --log-file /var/log/gunicorn.log aleph.manage:app
```

Comment:
We should use gevent and -w 1 (or 2). With the default worker type, -w 5 means we can handle 5 concurrent requests, which is pretty lame.